configure.ac

```diff
@@ -858,9 +858,9 @@ elif test x$with_runtime_preset = xfullaot; then
    mono_feature_disable_reflection_emit='yes'
    mono_feature_disable_appdomains='yes'
 
-   AOT_BUILD_FLAGS="-O=gsharedvt --aot=full,$INVARIANT_AOT_OPTIONS"
+   AOT_BUILD_FLAGS="--runtime=mobile -O=gsharedvt --aot=full,$INVARIANT_AOT_OPTIONS"
 
-   AOT_RUN_FLAGS="--full-aot"
+   AOT_RUN_FLAGS="--runtime=mobile --full-aot"
```

Comment:
@akoeplinger @marek-safar I don't understand what I'm doing.  But here's what I observe:

the fullaot preset seems to build the BCL libs with the strong name version of "2.0.5.0"  (this is coming from `mcs/build/common/Consts.cs.in`).

If I don't pass `--runtime=mobile` to the runtime, it consults the `framework_assemblies` table in appdomain.c and remaps requests for, for example, `System.Core 2.0.5.0` to `System.Core 4.0.0.0`.  So then my strong name check doesn't load the BCL assemblies from `mcs/class/lib/testing_aot_full` since they claim to be version 2.0.5.0.

If I pass `--runtime=mobile`, all those assemblies get mapped to 2.0.5.0 (ie the same as what gets built) and the tests run.

Is `configure.ac` the right place to do this?